go.mod

```diff
@@ -32,5 +32,6 @@ require (
 	github.com/tebeka/selenium v0.9.9
 	github.com/tstranex/u2f v1.0.0
 	github.com/valyala/fasthttp v1.15.1
+	golang.org/x/text v0.3.3
```

Comment:
Not sure this is required to be committed back, can you turn a quick `go mod tidy` to confirm?